.github/workflows/publish.yml

```diff
@@ -26,13 +28,26 @@ jobs:
               run: $(npm bin)/can-npm-publish
               continue-on-error: true
             - name: Publish
-              if: steps.check_availability.conclusion == success
+              id: npm_publish
+              if: steps.check_availability.outcome == "success"
               uses: JS-DevTools/npm-publish@v1
               with:
                   token: ${{ secrets.NPM_TOKEN }}
+            - name: Tagging
+              if: steps.npm_publish.conclusion == "success"
+              run: |
+                TAG=v$(jq -r ".version" package.json)
+                if git show-ref --tags | grep -q "refs/tags/${TAG}"
+                then
+                  echo "${TAG} exists. Skipped."
+                  exit
+                else
+                  git tag ${TAG}
+                  git push origin --tags
+                fi
```

Comment:
This is a nice work! We have four situations.

1. There is the tag and the version is avaiable.
2. There is not the tag and the version is avaiable.
3. There is not the tag and the version is unavailable.
4. There is the tag and the version is unavailable.

Your pull request makes us to publish it if the situation is 1 and 2.
However, I prefer to publish it if the situation is only 2.

Reply:
@tani 
OK. I will update the workflow as follows. 

1. Check `can-npm-publish`.
2. Check that the tag is in the repository.
3. Run `npm publish` If step 1 success and step 2 says it does not exist.
4. Run `git tag` if step 3 success.

In order to avoid only one of them succeeding, the execution condition for step 4 is whether or not step 3 succeeds.

Is it OK?

Reply:
In my sense, the correspondence between tags and published versions is important.
It seems follow my sence. Excellent!

Reply:
I fixed it. Please confirm!